src/quo/components/wallet/address_text/view.cljs

```diff
@@ -8,26 +9,37 @@
             [utils.address :as utils]))
 
 (defn- colored-network-text
-  [theme network]
+  [{:keys [theme network size weight]}]
   (let [{:keys [network-name short-name]} network]
     [text/text
-     {:size  :paragraph-2
-      :style {:color (colors/resolve-color network-name theme)}}
-     (str short-name ":")]))
+     {:size   size
+      :weight weight
+      :style  {:color (colors/resolve-color (or network-name (keyword network)) theme)}}
```

Comment:
btw `(keyword network)` looks suspicious since `network` is a map, no?

Reply:
right, but you are also destructuring `network` up above
```
(let [{:keys [network-name short-name]} network
```
so then it's strange that it's a string or a map, probably shouldn't destructure the string in the case that's what it is 🤔

Reply:
But you're destructing here which may be a string here also

Reply:
Done